hugo_site/themes/dceu2019/static/css/style.css

```diff
@@ -1,23 +1,32 @@
-body, html
-{
+body, html {
   width: 100%;
   padding: 0;
   margin: 0;
 }
+
 body {
   font-family: "Courier New", monospace;
-  font-size: 130%;
 }
 
 h1 {
-  font-size: 250%;
   text-align: center;
 }
 
 .header_image {
   text-align: center;
 }
 
+.header_image a {
+  max-width: 600px;
+  margin: 0 auto;
+  display: block;
+}
+
+.header_image img {
+  width: 100%;
+  margin: 0 auto;
+}
```

Comment:
This part makes the header image stretch beyond its size. Notice the conditional statement in the template that makes the logo big on the front page and smaller on the other pages.

![image](https://user-images.githubusercontent.com/374612/50566227-84917b80-0d37-11e9-93f3-d9d8e221e7d7.png)